db/migrate/20231005183849_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :email
+      t.string :photo
+
+      t.timestamps
+    end
```

Comment:
- [ ] Well done implementing all the models however according to the ERD diagram, the user should have a name, photo, bio and posts counter:


`create_table :users do |t|
    t.string :name
    t.string :photo 
    t.text :bio
    t.integer :posts_counter
    t.timestamps 
  end
`